walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js

```diff
@@ -12,21 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Generates a URL for a random image in the images directory and adds an img
- * element with that URL to the page.
- */
-function randomizeImage() {
-  // The images directory contains 13 images, so generate a random index between
-  // 1 and 13.
-  const imageIndex = Math.floor(Math.random() * 13) + 1;
-  const imgUrl = 'images/stanley-' + imageIndex + '.jpg';
+$(document).ready(function(){
 
-  const imgElement = document.createElement('img');
-  imgElement.src = imgUrl;
+    
+    $('a[href="#home"]').click(function() {
+       $('html,body').animate({
+           scrollTop: $(".home").offset().top},
+          2000);
```

Comment:
I see this function to animate reused 4 times in this file. I think following the DRY principle and creating a utility function to animate might clean things up a bit.

```
const animateScrollTop = (targetClass) => {
  $("html").animate({ scrollTop: $("." + targetClass).offset().top }, 2000);
};

// From each click listener, call animateScrollTop().
```

Reply:
done